accepted/2024/improve-support-for-packages-signed-with-trusted-signing.md

```diff
@@ -0,0 +1,155 @@
+# Improve support for NuGet packages that are signed with a Trusted Signing certificate
+
+- Author Name: Dirk Lemstra [dlemstra](https://github.com/dlemstra)
+- GitHub Issue: https://github.com/NuGet/NuGetGallery/issues/10027
+
+## Summary
+
+This proposal is about improving the support for NuGet packages that are signed with a Trusted Signing certificate. This proposal
+consists of three steps to improve the experience for users that sign their packages with Trusted Signing. These three steps can
+be released separately but I can also understand that the NuGet team would want to launch this as a complete feature.
+
+## Motivation
+
+Earlier this year [Trusted Signing](https://learn.microsoft.com/en-us/azure/trusted-signing/) was launched by Microsoft and 
+recently support for this was added to `dotnet sign`. Packages signed with this can be uploaded to the NuGet Gallery. But
+because the certificate is only valid for three days a user will probably always need to update their certificate on their 
```

Comment:
It's three days now but it could be shorter--do we want to say a specific number or say "short".

Reply:
I have updated it to "a short time" now.